src/pages/editor/PublisherOwnershipSettings.tsx

```diff
@@ -1,14 +1,27 @@
 import { Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
+import { fetchClaimedChannels } from '../../api/customerInstitutionsApi';
+import { ChannelClaimTable } from './ChannelClaimTable';
+import { PublicationChannelType } from '../../types/registration.types';
 
 export const PublisherOwnershipSettings = () => {
   const { t } = useTranslation();
 
+  const channelClaimsQuery = useQuery({
+    queryKey: ['channelClaims'],
+    queryFn: fetchClaimedChannels,
+  });
+
+  const channelClaims = channelClaimsQuery.data?.channelClaims;
   return (
     <>
       <Typography variant="h1" gutterBottom>
         {t('editor.institution.administer_publisher_channel_ownership')}
       </Typography>
+      {channelClaims && channelClaims.length > 0 && (
+        <ChannelClaimTable channelClaimList={channelClaims} channelType={PublicationChannelType.Publisher} />
```

Comment:
Naturlig at disse har samme navn?
```suggestion
        <ChannelClaimTable channelClaims={channelClaims} channelType={PublicationChannelType.Publisher} />
```